### ISSUE TITLE
Error on 'unicode' method in Python 3

### ISSUE THREAD

**Issue by tim1234567** [cloudinary/pycloudinary#49]
Error **name 'unicode' is not defined** occurs in Python 3.

File: templatetags/cloudinary.py
Line: 43
```
38 @register.simple_tag
39 def cloudinary_direct_upload_field(field_name="image", request=None):
40     form = type("OnTheFlyForm", (Form,), {field_name : CloudinaryJsFileField() })()
41     if request:
42         cl_init_js_callbacks(form, request)
43     return unicode(form[field_name])
```

**Comment by TalLevAmi** [comment]
Solved by 255093c55f4cd6aee1fd059e2fcb651b81560f9d
Please try and see if it works for you? 
A package will be released in the next few days.

**Comment by tim1234567** [comment]
Yep, it works now! Thank you a lot!
